tokens/delivery/http/tokens_delivery.go

```diff
@@ -163,11 +171,18 @@ func (a *TokensHandler) GetPrices(c echo.Context) (err error) {
 		}
 	}
 
-	prices, err := a.TUsecase.GetPrices(ctx, baseDenoms, []string{defaultQuoteChainDenom}, domain.ChainPricingSourceType)
+	// Check if pricingSource param is valid
+	pricingSourceInt, err := strconv.Atoi(c.QueryParam("pricingSource"))
```

Comment:
Could we do stricter validation here?

For example, if a user tries to provide a type but misconfigures due to a typo, we should fail the request.

So I see the following cases:
- "" -> chain source
- 0 -> chain source
- 1 -> coingecko source
- anything else -> error